src/tools/rbd/action/Device.cc

```diff
@@ -119,7 +131,7 @@ void add_device_type_option(po::options_description *options) {
 #ifdef _WIN32
      "device type [wnbd]");
 #else
-     "device type [ggate, krbd (default), nbd]");
+     "device type [ggate, krbd (default), nbd, ubbd]");
```

Comment:
Need to update `src/test/cli/rbd/help.t` accordingly.

Reply:
thanx